api/internal/handler/data_loader/route_import_test.go

```diff
@@ -23,7 +23,8 @@ import (
 	"io/ioutil"
 	"mime/multipart"
 	"net/http"
-	"os/exec"
+	"os"
+	"path"
```

Comment:
Do we need to consider separate path for Windows? Ref  https://pkg.go.dev/path#pkg-overview, the `path` package does not support Windows path.
```
 This package does not deal with Windows paths with drive letters or backslashes;
```

Reply:
sorry, use filepath instead, thks for your check

Reply:
@zaunist backend-e2e-test error,  [click](https://github.com/apache/apisix-dashboard/runs/4756297493?check_suite_focus=true)

can you check it and rerun? thks